build/csv/csv-gen.sh

```diff
@@ -37,8 +39,37 @@ ROOK_CSIADDONS_IMAGE=${ROOK_CSIADDONS_IMAGE:-${LATEST_ROOK_CSIADDONS_IMAGE}}
 # FUNCTIONS #
 #############
 
+function install_operator_sdk() {
+
+    local platform="${OS}_${PLATFORM}"
+    local tools_dir="${CACHE_DIR:-${WORK_DIR:-/tmp}}/tools"
+    local operator_sdk="${tools_dir}/operator-sdk-${OPERATOR_SDK_VERSION}"
+
+    # Check if operator-sdk 1.25 is already present in PATH
+    if command -v operator-sdk >/dev/null 2>&1; then
+        if operator-sdk version 2>/dev/null | grep "1.25.0"; then
+            export OPERATOR_SDK="operator-sdk"
+            return
+        fi
+    fi
+
+    if [ ! -f "$operator_sdk" ]; then
+        echo "=== installing operator-sdk ${OPERATOR_SDK_VERSION}"
+        mkdir -p "$tools_dir"
+        curl -JL -o "$operator_sdk" \
+            "https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk_${platform}"
+        chmod +x "$operator_sdk"
+        $operator_sdk version
+    fi
+
+    export PATH="${tools_dir}:${PATH}"
```

Comment:
Can we just always set the var to be the full path to the operator sdk, instead of adding it to to the PATH?